lua/lspconfig/server_configurations/drools_lsp.lua

```diff
@@ -1,35 +1,60 @@
 local util = require 'lspconfig.util'
 
-local get = function(config, section, key)
-  return vim.tbl_get(config.settings, section, key) and config.settings[section][key] or nil
+local function get(config, segments)
```

Comment:
```lua
local function get(config, segments)
 local cfg = { }
  if vim.tbl_get(config, unpack(segments)) then
    vim.list_extend(cfg, segments)
  end
  return cfg
end
```

Reply:
@glepnir, I'm not tring to add those segments (which are node "path" keys) into a new table. I'm traversing down into the config table, and extracting the value (if it exists) of a node that matches the segment "path". So, `config` is a table of nested tables, and each value in `segments` is really just a key into each subsequent nested table. I don't want to add `segments` itself.